_posts/0000-01-02-mfwalchak.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Fear is the mindkiller. Fear is the little death that brings total obliteration.